src/ol_openedx_chat/README.rst

```diff
@@ -1,23 +1,81 @@
 ol-openedx-chat
 ###############
 
-An xBlock aside to add MIT Open Learning chat into xBlocks
+An xBlock aside to add MIT Open Learning chat into xBlocks.
 
 
 Purpose
 *******
 
 MIT's AI chatbot for Open edX
 
-Getting Started with Development
-********************************
+Setup
+=====
 
+1) Add OL chat as a dependency
+------------------------------
 
-Deploying
-*********
+For local development, you can use one of the following options to add
+this as a dependency in the ``edx-platform`` repo:
+
+-  **Install directly via pip.**
+
+   ::
+
+      # From the devstack directory, run bash in a running LMS container...
+      make dev.shell.lms
+
+      # In bash, install the package
+      source /edx/app/edxapp/edxapp_env && pip install ol-openedx-chat==<version>
+
+      # Do the same for studio
+      make dev.shell.studio
+
+      # In bash, install the package
+      source /edx/app/edxapp/edxapp_env && pip install ol-openedx-chat==<version>
+
+-  **Build the package locally and install it**
+
+   ::
+
+      Follow these steps in a terminal on your machine:
+
+      1. Navigate to open-edx-plugins directory
+
+      2. If you haven't done so already, run "./pants build"
+
+      3. Run "./pants package ::". This will create a "dist" directory inside "open-edx-plugins" directory with ".whl" & ".tar.gz" format packages for "ol_openedx_chat" and other "ol_openedx_*" plugins in "open-edx-plugins/src"
+
+      4. Move/copy any of the ".whl" or ".tar.gz" files for this plugin that were generated in the above step to the machine/container running Open edX (NOTE: If running devstack via Docker, you can use "docker cp" to copy these files into your LMS or CMS containers)
+
```

Comment:
Another option with the Devstack, if open-edx-plugins is in the src folder, we can go to shell and do `pip install /edx/src/open-edx-plugins/dist/ol-chat...tar.gz`